docs/resources/Channel.md

```diff
@@ -849,13 +849,15 @@ When setting `archived` to `false`, when `locked` is also `false`, only the `SEN
-| archived              | boolean  | whether the channel is archived                                                                                                                                                                   |
-| auto_archive_duration | integer  | duration in minutes to automatically archive the thread after recent activity, can be set to: 60, 1440, 4320, 10080                                                                               |
-| locked                | boolean  | when a thread is locked, only users with MANAGE_THREADS can unarchive it                                                                                                                          |
-| rate_limit_per_user   | ?integer | amount of seconds a user has to wait before sending another message (0-21600); bots, as well as users with the permission `manage_messages`, `manage_thread`, or `manage_channel`, are unaffected |
+| Field                   | Type     | Description                                                                                                                                                                                       |
+|-------------------------|----------|---------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|
+| name                    | string   | 2-100 character channel name                                                                                                                                                                      |
+| archived                | boolean  | whether the channel is archived                                                                                                                                                                   |
+| auto_archive_duration\* | integer  | duration in minutes to automatically archive the thread after recent activity, can be set to: 60, 1440, 4320, 10080                                                                               |
+| locked                  | boolean  | when a thread is locked, only users with MANAGE_THREADS can unarchive it                                                                                                                          |
+| rate_limit_per_user     | ?integer | amount of seconds a user has to wait before sending another message (0-21600); bots, as well as users with the permission `manage_messages`, `manage_thread`, or `manage_channel`, are unaffected |
+
+\* The 3 day and 7 day archive durations require the server to be boosted. The [guild features](#DOCS_RESOURCES_GUILD/guild-object-guild-features) will indicate if a server is able to use those settings.
```

Comment:
Can the value in minutes for 3 and 7 day be mentioned? Would be handy imo

Same as the other location